ttnn/cpp/ttnn/operations/data_movement/transpose/transpose.hpp

```diff
@@ -6,24 +6,95 @@
 
 #include "ttnn/run_operation.hpp"
 #include "ttnn/decorators.hpp"
-#include "ttnn/deprecated/tt_dnn/op_library/transpose/transpose_op.hpp"
+#include "device/transpose_op.hpp"
+#include "ttnn/operations/data_movement/permute/permute.hpp"
 
 
 namespace ttnn {
 namespace operations::data_movement {
 
 constexpr uint32_t TransposeDefaultQueueID = 0;
 
+inline Tensor transpose_(const Tensor &a, TransposeOpDim transpose_dim, const MemoryConfig& output_mem_config) {
```

Comment:
is this an in-plaice version?

Reply:
this is just a helper function only used within this file.